compiler/il/Aliases.cpp

```diff
@@ -136,6 +136,7 @@ OMR::SymbolReference::getUseonlyAliasesBV(TR::SymbolReferenceTable * symRefTab)
             case TR_overflowCheck:
             case TR_nullCheck:
             case TR_methodTypeCheck:
+            case TR_incompatibleReceiver:
```

Comment:
Perhaps we can guard them with `J9_PROJECT_SPECIFIC` to emphasize the fact that this is J9 specific and to make an easy eye catcher for future refactoring?